.github/workflows/ansible-test.yml

```diff
@@ -5,20 +5,24 @@ on:
 jobs:
   sanity:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python_version: ["3.6"]
+        ansible_version: ["2.9", "2.10"]
```

Comment:
Testing against `devel` is also a good idea, since it from time to time gets new tests. Then you won't get surprised (too much) when suddenly ansible-base 2.11 is released :)